db/migrate/20241029105239_create_tickets_edit_person.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CreateTicketsEditPerson < ActiveRecord::Migration[7.2]
```

Comment:
I'm not too wild about the repeated nature of `previous_*` and `new_*` fields.
Every time you want to extend/augment the capability of this ticket type in the future, you'd have to write a migration with two new columns.

Instead, what would you think about an auxiliary table along the lines of "EditPersonChanges", with a foreign key pointing to this table, and three columns `field_name`, `old_value`, `new_value`?

`field_name` can be enumized, to allow some degree of control over which fields are "allowed" to be edited, but at the same time also make future extensions very easy (just add an entry to the enum, which is much less invasive than a two column migration)

And if you want, you might even store the `old_value` and `new_value` as JSON to allow various field types. Just an idea :)